src/components/TableRowComponent.jsx

```diff
@@ -1,25 +1,38 @@
 import React from 'react';
+import _ from 'lodash';
 import PropTypes from 'prop-types';
 import { Table } from 'semantic-ui-react';
 
-const TableRowComponent = props => (
-  <Table.Row onClick={props.handleViewClick}>
-    {props.headings
-      .map(heading => (
-        <Table.Cell key={heading}>
-          {props.data[heading]}
-        </Table.Cell>
-      )
-      )}
-    {props.children}
-  </Table.Row>
-);
+class TableRowComponent extends React.Component {
```

Comment:
@Ewanjiru I can infer that you are passing `history` from the parent as prop to this component. You can use `withRouter` here, so as to avail the history prop to the component instead of depending on the parent. Therefore, the component will be exported as `export default withRouter(TableRowComponent)`. An example of this usage is in the [LoginComponent](https://github.com/AndelaOSP/art-dashboard/blob/7510fe1bcb62b86aaba3e7225b378c123e31581a/src/components/LoginComponent.jsx)

Reply:
Done. Thanks.

Reply:
As discussed, considering the TableRowComponent has been re-used in several components, wrapping it with the withRouter has led to an issue with the tests for all the related components. 
In this case, I will be reverting to using the history props from the parent but will be open for change once I find another solution to the tests.